vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/core/LayoutableShadowNode.cpp

```diff
@@ -151,8 +151,21 @@ LayoutMetrics LayoutableShadowNode::computeRelativeLayoutMetrics(
   }
 
   // ------------------------------
-
-  auto transformedFrames = calculateTransformedFrames(shadowNodeList, policy);
+  // TODO: T127619309 remove after validating that T127619309 is fixed
```

Comment:
Is this our TODO, or is it from the integrate?

Reply:
T127619309 looks like a meta task reference, so I assume its their todo.